packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/PBSProgressListenerTests.kt

```diff
@@ -1,35 +1,20 @@
-/*
```

Comment:
Removed copyright!?

Reply:
Not sure at what point it got removed. I have added it back.